content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -0,0 +1,171 @@
+through the smart button in the upper-right corner.
+
+.. image:: loyalty_discount/price-programs-items.png
+   :align: center
+   :alt: Program items tab
+
+.. _pricing_management/program-types:
+
+Program types
+-------------
+
+The different :guilabel:`Program Types` are:
+
+- :guilabel:`Coupons`: Generate and share single-use coupon codes that grant immediate access to
+  rewards.
+- :guilabel:`Next Order Coupons`: Generate and share single-use coupon codes that grant access to
+  rewards on the next order.
+- :guilabel:`Loyalty Cards`: When making purchases, the customer accumulates points to exchange for
+  rewards on future orders.
+- :guilabel:`Promotions`: Set conditional rules for ordering products, which, when fulfilled, grant
+  access to rewards for the customer.
+- :guilabel:`Discount Code`: Set codes which, when entered upon checkout, grant discounts to the
+  customer.
+- :guilabel:`Buy X Get Y`: For every X item bought, the customer is granted 1 credit. After
+  accumulating a specified amount of credits, the customer can trade them in to receive Y item.
+
+Conditional rules
+-----------------
+
+Next, configure the :guilabel:`Conditional rules` that determine when the program applies to a
+customer's order.
+
+In the :guilabel:`Rules & Rewards` tab, click :guilabel:`Add` next to :guilabel:`Conditional rules`
+to add *conditions*.
+
+.. image:: loyalty_discount/price-conditional-rewards.png
+   :align: center
+   :alt: Conditional rewards
+
+.. note::
+   The options for :guilabel:`Conditional rules` vary depending on the selected :ref:`Program Type
+   <pricing_management/program-types>`.
+
+The following options are available for configuring conditional rules:
+
+- :guilabel:`Discount Code`: Enter a custom code used for the :guilabel:`Discount Code` program, or
+  use the default one generated by Odoo.
+- :guilabel:`Minimum Quantity`: Enter the minimum number of products that must be purchased in order
+  to access the reward. Set the minimum quantity to at least `1` to ensure that the customer must
+  make a purchase in order to access the reward.
+- :guilabel:`Minimum Purchase`: Enter the minimum amount (in currency), with :guilabel:`tax
+  Included` or :guilabel:`tax Excluded`, that must be spent in order to access the reward. If both a
+  minimum quantity *and* minimum purchase amount are entered, then the customer's order must meet
+  both conditions.
+- :guilabel:`Products`: Select the product(s) for which the program applies.
+- :guilabel:`Categories`: Select the category of products for which the program applies. Choose
+  :guilabel:`All` to apply it to all categories.
+- :guilabel:`Product Tag:` Select a tag to apply the program to products with that tag.
+- :guilabel:`Grant`: Enter the number of points the customer earns :guilabel:`per order`,
+  :guilabel:`per currency spent`, or :guilabel:`per unit paid` (for the :guilabel:`Loyalty Cards`
+  and :guilabel:`Buy X Get Y` programs).
+
```

Comment:
Perhaps it would be useful to mention the "Save & Close" and "Save & New" button options available on the pop-up...What do you think?